frontend/ui/widget/checkbutton.lua

```diff
@@ -70,7 +70,7 @@ function CheckButton:initCheckButton(checked)
     self._textwidget = TextBoxWidget:new{
         text = self.text,
         face = self.face,
-        width = (self.width or self.parent._input_widget.width) - self._checkmark.dimen.w,
+        width = (self.width or self.parent.added_widgets_width) - self._checkmark.dimen.w,
```

Comment:
No risk for this widget to be used by other `self.parent` widget than the 2 ones you touch in this PR, and get an "attempt to substract from a nil value") ?

Reply:
My mistake, I will re-check all the modules.